### DISCUSSION TITLE
Math shortcuts in op search are broken

### DISCUSSION THREAD

**Discussion by k0mrad** [cables-gl/cables_docs#884] [! Welcome - readme first!]
Until yesterday, you could enter *3 or /5 or +1 to create a math operator. It seems that now you need to add a Space.

**Comment by 60-hz** [comment]
I confirm.

**Comment by pandrr** [comment]
thanks for reaporting and creating an issue, yes its a bug, works again on dev.cables.gl, will hotfix on live later today or tomorrow morning
